app/views/candidate_interface/shared/_details.html.erb

```diff
@@ -1,5 +1,7 @@
 <% content_for :title, page_title %>
 
+<%= @application_form_presenter.application_form.candidate.id %>
```

Comment:
Is this intended?

Reply:
You're a star!!! No that was not intended! 😰

Reply:
Removed from original commit and force pushed 👌